.circleci/config.yml

```
@@ -20,7 +20,8 @@ jobs:
       - run: bash circle-test.sh
 
   test-influxdb:
-    machine: true
+    machine:
+      image: ubuntu-2004:202111-01
```

```suggestion
      image: ubuntu-2004:202111-02
```
There is evidently a newer version with log4j updates: https://circleci.com/docs/2.0/configuration-reference/#available-machine-images